packages/cosmic-swingset/provisioning-server/src/ag_pserver/main.py

```diff
@@ -309,6 +313,15 @@ def built(code):
         d.addErrback(log.err)
         return server.NOT_DONE_YET
 
+class GenesisJSON(resource.Resource):
+    def __init__(self, o):
+        self.opts = o
+
+    def render_GET(self, req):
+        f = open(cosmosGenesisFile(self.opts['home']))
```

Comment:
might use the context-manager idiom here:

```python
    with open(...) as f:
        config = f.read()
```

so the filehandle gets closed promptly